File: Bexchange.Domain/RequestModels/ChangeUserInfoRequest.cs
using BexchangeAPI.Domain.Models;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bexchange.Domain.RequestModels
{
    public class ChangeUserInfoRequest
    {
        public int Id { get; set; }
        public string UserName { get; set; }        
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public AddressInfo Address { get; set; }
    }
}

File: Bexchange.App/src/app/models/ChangeUserInfoRequest.ts
import { AddressInfo } from './AddressInfo';

export interface ChangeUserInfoRequest {
    id: number | null | undefined;
    userName: string | null | undefined;
    firstName: string | null | undefined;
    lastName: string | null | undefined;
    email: string | null | undefined;   
    address: AddressInfo | null | undefined;
}